Arm/rover_arm_system.hpp

```diff
@@ -121,129 +121,100 @@ class RoverArmSystem : public sjsu::common::RoverSystem
 
     HomeWrist();
   }
 
-  void HomeShoulder(Accelerometer::Acceleration_t rotunda_acceleration,
-                    Accelerometer::Acceleration_t shoulder_acceleration)
+  void HomeShoulder(Joint::Acceleration rotunda_acceleration,
+                    Joint::Acceleration shoulder_acceleration)
   {
-    double gravity                = 9.81;
-    double rotunda_acceleration_x = static_cast<double>(rotunda_acceleration.x);
-    double rotunda_acceleration_y = static_cast<double>(rotunda_acceleration.y);
-    double shoulder_acceleration_x =
-        static_cast<double>(shoulder_acceleration.x);
-    double shoulder_acceleration_y =
-        static_cast<double>(shoulder_acceleration.y);
-    if (rotunda_acceleration_x == 0.0)
+    // Make this into a helper function, this checks to see if any of the acceleration values are 0, and if they are it will change
+    // to something close to 0
+    double gravity = 9.81;
+    if (rotunda_acceleration.x == 0.0)
     {  // catches division by 0
-      rotunda_acceleration_x = .0001;
+      rotunda_acceleration.x = .0001;
     }
-    if (rotunda_acceleration_y == 0.0)
+    if (rotunda_acceleration.y == 0.0)
     {  // catches division by 0
-      rotunda_acceleration_y = .0001;
+      rotunda_acceleration.y = .0001;
     }
-    if (shoulder_acceleration_x == 0.0)
+    if (shoulder_acceleration.x == 0.0)
     {  // catches division by 0
-      shoulder_acceleration_x = .0001;
+      shoulder_acceleration.x = .0001;
     }
-    if (shoulder_acceleration_y == 0.0)
+    if (shoulder_acceleration.y == 0.0)
     {  // catches division by 0
-      shoulder_acceleration_y = .0001;
+      shoulder_acceleration.y = .0001;
     }
+//TODO: clean up calculations into helper functions to make it more legible
     double rotunda_angle =
-        acos(gravity / rotunda_acceleration_y) *  // for angle
-        (rotunda_acceleration_x /
-         fabs(rotunda_acceleration_x));  // for direction
+        acos(gravity / rotunda_acceleration.y) *  // for angle
+        (rotunda_acceleration.x /
+         fabs(rotunda_acceleration.x));  // for direction
     double shoulder_angle =
-        acos(gravity / shoulder_acceleration_y) *  // for angle
-        (shoulder_acceleration_x /
-         fabs(shoulder_acceleration_x));  // for direction
-    units::angle::degree_t home =
-        static_cast<units::angle::degree_t>(rotunda_angle + shoulder_angle);
-    rotunda_.SetPosition(home);
+        acos(gravity / shoulder_acceleration.y) *  // for angle
+        (shoulder_acceleration.x /
+         fabs(shoulder_acceleration.x));  // for direction
+    double home = rotunda_angle + shoulder_angle;
+    MoveShoulder(10.0, home);
   }
 
-  void HomeElbow(Accelerometer::Acceleration_t rotunda_acceleration,
-                 Accelerometer::Acceleration_t elbow_acceleration)
+  void HomeElbow(Joint::Acceleration rotunda_acceleration,
+                 Joint::Acceleration elbow_acceleration)
```

Comment:
Nitpick - might be nice to have this under the same namespace somehow. Seeing Joint:: everywhere doesn't look nice

Reply:
I'll see what I can do.